types/office-js/index.d.ts

```diff
@@ -10153,7 +10153,7 @@ declare namespace Office {
          *        asyncContext: Developers can provide any object they wish to access in the callback method.
          * @param callback Optional. When the method completes, the function passed in the callback parameter is called with a single parameter of type Office.AsyncResult. 
          */
-        saveAsync(options?: Office.AsyncContextOptions, callback?: (result: AsyncResult<void>) => void): void;
+        saveAsync(options?: Office.AsyncContextOptions, callback?: (result: AsyncResult<string>) => void): void;
```

Comment:
Above this is a comment "`saveAsync(callback: (result: AsyncResult<void>) => void): void;`" - can you update from <void> to <string> as well? Thanks.

Reply:
Thank you, I missed that, fixed now